pkgs/development/python-modules/pynanoleaf/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, requests }:
+
+buildPythonPackage rec {
+  pname = "pynanoleaf";
+  version = "0.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2ced000e3c37f4e2ce0ea177d924af71c97007de9e4fd0ef37dcd7b4a6d1b622";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [ requests ];
+
+  meta = with lib; {
+    homepage = https://github.com/Oro/pynanoleaf;
```

Comment:
rfc 45
```suggestion
    homepage = "https://github.com/Oro/pynanoleaf";
```

Reply:
Thanks for the review! Force pushed the suggested change